clickControl/clickControl.py

```diff
@@ -78,50 +129,57 @@ def startClick(self, msg, userMode=False, DPDK=True):
 
             (output, err) = execl.execAndRead("sudo click-install -j 2 %s" % self.click_config)
             if err != "":
-                self.log.error("Click: %s" % err)
+                self.log.error("Click: %s", err)
                 return False
 
         else:  # not in kernel - does not daemonize itself, so we handle it as a proc.
-            if DPDK:
+            if dpdp:
                 cmd = 'click  --dpdk -c 0xffffff -n 4 -- -u /click {}'.format(self.click_config)
-            elif userMode:
+            elif user_mode:
                 cmd = 'click {} -u /click'.format(self.click_config)
             else:
-                log.error('startClick must be one of kernel, userMode, or DPDK')
+                self.log.error('startClick must be one of kernel, user_mode, or dpdp')
 
-            self.log.info('Running cmd: {}'.format(cmd))
-            self._clickProc = Popen(cmd.split())
+            self.log.info('Running cmd: %s', cmd)
```

Comment:
I prefer "".format() but that is a small thing.

Reply:
pep8 doesnt want you to use .format with logs

best source with an explanation: https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation

Reply:
If you are in favor, I can add logging-format-interpolation to the disable list for warnings

Reply:
Oh, no. Not using format() makes sense now. I figured there was a good reason.

Reply:
Its awkward, but I prefer format for everything, except for logging module.